CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Application Server pub/sub integrations race condition during shutdown.
 - Console webhook templates empty headers error.
 - Console MQTT URL validation.
+- Password updates not working in the OAuth frontend.
```

Comment:
This doesn't need to be in changelog, since it fixes something that we broke in this version, not something that was already broken in a previous version.